build-tools/packages/build-cli/docs/bump.md

```diff
@@ -63,15 +63,15 @@ Update the dependency version of a specified package or release group. That is,
 ```
 USAGE
   $ flub bump deps PACKAGE_OR_RELEASE_GROUP [-v] [--prerelease -t
-    latest|newest|greatest|minor|patch|@next|@canary] [--onlyBumpPrerelease] [-g client|server|azure|build-tools | -p
-    <value>] [-x | --install | --commit |  |  | ]
+    latest|newest|greatest|minor|patch|@next|@canary] [--onlyBumpPrerelease] [-g
+    client|server|azure|build-tools|gitrest|historian | -p <value>] [-x | --install | --commit |  |  | ]
```

Comment:
We might want to remove the azure as it is part of client release group now

Reply:
Agreed. I'm leaving that for a future PR once the release group definitions have settled a bit.